api/models/entity/Activity.ts

```diff
@@ -5,27 +5,29 @@ import {
     ManyToOne,
     BeforeInsert,
     BeforeUpdate,
-    CreateDateColumn, getConnection, OneToMany
+    CreateDateColumn,
+    getConnection,
+    OneToMany
 } from 'typeorm';
 import { Government, IGovernmentSummary } from './Government';
 import { Campaign, ICampaignSummary } from './Campaign';
 import { IUserSummary, User } from './User';
 import { IsDefined, validate, ValidationError } from 'class-validator';
 import { IUserPermission, IUserPermissionResult } from './Permission';
 import { Contribution } from './Contribution';
-import {Expenditure} from "./Expenditure";
+import { Expenditure } from './Expenditure';
 
 export enum ActivityTypeEnum {
     USER = 'user',
     PERMISSION = 'permission',
     CAMPAIGN = 'campaign',
     GOVERNMENT = 'government',
     INVITATION_EMAIL = 'invitation email',
+    CONTRIBUTION = 'contribution'
```

Comment:
@danmelton for create/update/archive contribution events, should the activity type be 'contribution' or 'user'?